src/main/resources/templates/requests/provider-request-detail.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en" th:replace="~{layout :: layout(title=~{::title},mainContent=~{::#main-content},pageCategory=${'actions'},breadcrumbs=~{::#breadcrumbs})}"
       xmlns:th="http://www.thymeleaf.org" xmlns:moj="http://www.thymeleaf.org">
 <head>
-  <title>LAA CCMS - Provider request</title>
+  <title th:text="#{request.title}"></title>
```

Comment:
A text property already exists for the service name, so this can be split into two properties, e.g.

```suggestion
  <title th:text="|#{service.name} - #{request.title}|"/>
```

The same goes for the titles of the other pages in this PR.